CAM/SetTableGamesCameraThisUpdate.md

```diff
@@ -0,0 +1,40 @@
+---
+ns: CAM
+aliases: ["_0x79C0E43EB9B944E2"]
```

Comment:
Remove the underscore in front of the native hash

Reply:
Fixed!